libs/tup-components/src/tickets/TicketsTable.global.css

```diff
@@ -0,0 +1,15 @@
+.ticket-reply-required {
+  font-weight: 500;
+}
+
+.ticket-unknown-status {
+  font-family: monospace;
```

Comment:
**Opportunity for Existing Variable:**
```suggestion
  font-family: var(--global-font-family--mono);
```

Design has yet to define [our monospace font](https://github.com/TACC/Core-Styles/blob/22783d8/src/lib/_imports/settings/font.css#L29). They will. I anticipate it will be Roboto Mono.